sonar-project.properties

```diff
@@ -0,0 +1,2 @@
+# Disable specific duplicate code since it would introduce more complexity to reduce it.
+sonar.cpd.exclusions=src/standard.ts
```

Comment:
hmm, is this line wrong? :thinking:

Reply:
@aayushmau5 Could you test with `sonar.exclusions` not `sonar.cpd.exclusions`?

Reply:
checking it.

Reply:
no luck.